Cargo.toml

```diff
@@ -17,4 +17,10 @@ uuid = { version = "0.8.1", features = ["serde", "v4"] }
 
 [dev-dependencies]
 quickcheck = "0.9.2"
-quickcheck_macros = "0.9.1"
\ No newline at end of file
+quickcheck_macros = "0.9.1"
+
+[profile.release]
+
+# Turns Link-Time Optimization up to the MAX! Results in slower
+# release build times, but faster and significantly smaller binaries.
+lto = true
```

Comment:
BOOM!  I really like LTO :) 👍

Reply:
With this off, it draws the boundaries for LTO at a crate level. With it on, it does the LTO on the whole resulting binary. That's why the binaries are so much smaller, it can know for sure that some code will never be used and it can strip it out.